edgelet/iotedged/src/error.rs

```diff
@@ -54,6 +54,20 @@ pub enum ErrorKind {
     Parse,
     #[fail(display = "An http error occurred.")]
     Http,
+    #[cfg(target_os = "windows")]
+    #[fail(
+        display = "Edge device information is required.\n\
+        Please edit your config.yaml and provide information to connection the Edge to the IoT Hub.\n\
```

Comment:
"to connect the Edge device to IoTHub. "
Or even better, "Please update the config.yaml and provide the IoTHub connection information."